sites/all/themes/orwell/sass/bootstrap/_grid.scss

```diff
@@ -3,32 +3,35 @@
 
 $grid-columns: 12;
 
+// Using ems for media queries, as they're the most consistent:
+// https://zellwk.com/blog/media-query-units/
```

Comment:
this seems like quite the upgrade.. have you checked the site through on various browsers, to see that theres no elements that breaks cause of this?

Reply:
also, it seems like the rest of the codebase still uses REM, so it seems kinda problematic to mix both? what is the issue you're trying to fix here? :)

Reply:
Didn't read the link, eh?

Well the short story is that em makes the mediaqueries Do The Right Thing™ regardless of zoom, user changing default font-size, etc.

If the user hasn't changed default font-size, using ems instead of rems changes nothing. So we're fixing something we weren't testing for.

Reply:
I know :)
The thing is that EM and REM are designed to do different things, and I'm just wondering if this will break a ton of things cause its a mix of those two things now

Assuming you've clicked around and checked in various browsers, i approve this :)